src/app/api/auth/[auth0]/route.js

```diff
@@ -0,0 +1,3 @@
+import { handleAuth } from '@auth0/nextjs-auth0';
+
+export const GET = handleAuth();
```

Comment:
Update `README.md` with developer instructions instructions that `.env.local` need to be setup with auth0 configuration as given in Auth0 documentation URL.